mobile/src/modules/auth/store/reducer.ts

```diff
@@ -0,0 +1,46 @@
+import { Account } from '@burstjs/core';
+import { Reducer } from '../../../core/interfaces';
+import { createReducers } from '../../../core/utils/store';
+import { actionTypes } from './actionTypes';
+
+export interface AuthReduxState {
+  accounts: Account[];
+}
+
+export const authState = (): AuthReduxState => {
+  return {
+    accounts: []
+  };
+};
+
+const addAccount: Reducer<AuthReduxState, Account> = (state, action) => {
+  const accounts = [ ...state.accounts, action.payload ];
+  return {
+    ...state,
```

Comment:
bit of boilerplate. Note that this is a shallow copy only. Maybe worth introducing a state merge function, like `updateState(state, obj}` or even use stuff like [seamless-immutable](https://github.com/rtfeldman/seamless-immutable)

Reply:
`updateState` can be `merge` from lodash for example